src/components/Gallery/_gallery.scss

```diff
@@ -41,17 +41,56 @@
 .cc-gallery__media-item {
   @extend %flex-grid-item;
   @include flex-grid-item(2);
-  appearance: none;
-  background: transparent;
-  border: 0;
   
   @include mq(sm) {
     @include flex-grid-item(3);
   }
 }
 
+.cc-gallery__media-frame {
+  appearance: none;
+  background: var(--colour-grey-50);
+  border: 0;
+  color: var(--colour-white);
+  cursor: pointer;
+  display: block;
+  height: 100%;
+  outline: 0;
+  position: relative;
+  width: 100%;
```

Comment:
Presuming lots of this is resetting <button> styles? I'd probably abstract those styles into some kind of btn-reset mixin/extend/util to make it re-usable, but not hugely important

Reply:
indeed :)